scripts/generate/templates/SafeCast.js

```diff
@@ -155,6 +191,7 @@ function toUint${length}(int${length} value) internal pure returns (uint${length
 module.exports = format(
   header.trimEnd(),
   'library SafeCast {',
+  [...LENGTHS.map(toUintDownCastErrors), toUintErrors(256), ...LENGTHS.map(toIntDownCastErrors), toIntErrors(256)],
   [...LENGTHS.map(toUintDownCast), toUint(256), ...LENGTHS.map(toIntDownCast), toInt(256)],
```

Comment:
IMO that would be more readable.
```suggestion
  [
      ...LENGTHS.map(toUintDownCastErrors), 
      toUintErrors(256), 
      ...LENGTHS.map(toIntDownCastErrors), 
      toIntErrors(256)
      ...LENGTHS.map(toUintDownCast), 
      toUint(256), 
      ...LENGTHS.map(toIntDownCast), 
      toInt(256)
  ],
```

Reply:
(not sure what the linter has to say about this)